docs/how-to-guides/convert-changelog-to-table.md

```diff
@@ -10,90 +10,82 @@ keywords: changelog, table, materialize
 
 ## Context
 
-You have a stream of events that represent a series of changes, known as a changelog. You want a view of the data that reflects only the last change for each key. Because ksqlDB represents change over time using tables, you need a way to convert your changelog stream into a table. This is broadly called *materializing* a changelog stream into a table.
+You have a stream of events that represent a series of changes, known as a changelog. You want a view of the data that reflects only the last change for each key. Because ksqlDB represents change over time using tables, you need a way to convert your changelog into a table. This is broadly called *materializing* a changelog into a table.
```

Comment:
```suggestion
You have a topic that represent a series of changes, known as a changelog. You want a view of the data that reflects only the last change for each key. Because ksqlDB represents change over time using tables, you need a way to convert your changelog into a table. This is broadly called *materializing* a changelog into a table.
```

Reply:
> Because ksqlDB represents change over time using tables, you need a way to convert your changelog into a table

This sentence seems miss-leading... if you have a changlog in a topic, you can just call `CREATE [SOURCE] TABLE` to read it.